content/desktop/release-notes.md

```diff
@@ -21,6 +21,59 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.33.0
+
+{{< release-date date="2024-07-25" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.33.0" build_path="/TBD/" >}}
+
+### New
+
+- Docker Debug in now generally available.
+- BuildKit now evaluates Dockerfile rules to inform you of potential issues.
+- **Resource Allocation** settings can now be accessed directly from the resource usage data displayed in the Dashboard footer.
+
```

Comment:
Please add:

- Compose Bridge command line option is now available via Experimental features. When enabled, run `compose-bridge` to convert your Compose configuration to Kubernetes resources.